sdcm/cluster.py

```diff
@@ -159,12 +159,23 @@ def register_cleanup(cleanup='destroy'):
 
 
 class Setup(object):
+
+    _status = {
```

Comment:
This is tricky logic instead of using If statement. Remember the KISS principle ;)
So please or hide the logic inside the class method

Reply:
Fix